subset/bacnet/picstest/test_bacpics

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+JAR=bacnet4j/bacnet4j-8jdk.jar
+CLASS=com.serotonin.bacnet4j.test.RedstoneTest.Test5
```

Comment:
need descriptive name, not "RedstoneTest" (organization) or "Test5" -- that has no meaning.